package.json

```diff
@@ -38,17 +41,20 @@
     "uuid": "^8.3.2"
   },
   "devDependencies": {
+    "@types/cookie-parser": "~1.4.6",
     "@types/express": "^4.17.13",
     "@types/jest": "^27.4.1",
     "@types/node": "^18.11.18",
     "@types/nunjucks": "^3.2.1",
+    "@types/sinon": "~7.5.2",
```

Comment:
ditto

Reply:
thanks @JoelClemence I have modified it.